### PR TITLE
Fix the plain text URL for invitations

### PR DESCRIPTION
**Pull request by m-terlinde**

### CODE REVIEW

**Comment by czue** [comment]
thanks!

File: guests/invitation.py [hunk 1]
```diff
@@ -49,7 +49,7 @@ def send_invitation_email(party, test_only=False, recipients=None):
     context['couple'] = settings.BRIDE_AND_GROOM
     template_html = render_to_string(INVITATION_TEMPLATE, context=context)
     template_text = "You're invited to {}'s wedding. To view this invitation, visit {} in any browser.".format(
-        settings.BRIDE_AND_GROOM,
+        settings.BRIDE_AND_GROOM, settings.WEDDING_WEBSITE_URL +
```
Comment (czue) [inline guests/invitation.py]: I probably would have put this on the same line as the `reverse` for clarity. Not a big deal, but I'll give you a chance to respond if you agree it's better.

**Review by czue — APPROVED** [review]

File: guests/invitation.py (hunk 1 above)
Comment (m-terlinde) [reply guests/invitation.py]: I totally blame it on autoformat and not on laziness :D.

File: guests/invitation.py [hunk 2]
```diff
@@ -49,8 +49,8 @@ def send_invitation_email(party, test_only=False, recipients=None):
     context['couple'] = settings.BRIDE_AND_GROOM
     template_html = render_to_string(INVITATION_TEMPLATE, context=context)
     template_text = "You're invited to {}'s wedding. To view this invitation, visit {} in any browser.".format(
-        settings.BRIDE_AND_GROOM,
-        reverse('invitation', args=[context['invitation_id']])
+        settings.BRIDE_AND_GROOM,
```
Comment (czue) [inline guests/invitation.py]:
```suggestion
        settings.BRIDE_AND_GROOM,
```

**Pull request merged**